src/app/public/modules/autonumeric/autonumeric-config.ts

```diff
@@ -0,0 +1,6 @@
+export class AutonumericConfig {
+  public constructor(
+    public preset?: string,
```

Comment:
Just to make things a bit more clear for consumers, could we rename this to `languagePreset`?

Reply:
Updated this to `languagePreset`.  I also updated the corresponding `@Input` in the directive.